README.md

```diff
@@ -61,7 +61,7 @@ You can change the HTML/CSS layout if you need it.
 ## Deploy and Pull Request
 
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_2048_game/)
+    - [DEMO LINK](https://iLavruk.github.io/js_2048_game/)
```

Comment:
Add this demo link to the PR description 
<img width="984" alt="image" src="https://github.com/user-attachments/assets/e0a3c59f-710f-4f65-a11a-582e1c7518f6" />